doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
+aliases: "/telemetry/cli"
+---
+
+The Things Stack\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used. The objective with the introduction of telemetry is to be able to better understand what is used so that we can expand and improve on the experience that we provide to our users.
+
+## Identification
+
+The worry that data is attached to an easily identifiable piece of information is common and for good reason, therefore we believe that it is the first thing we should address.
+
+When the CLI start the procedure to collect and send the telemetry data for the first time, it creates a file on the system's cache folder which contains the following data: 
+```yaml
+uid: 00000000-0000-0000-0000-000000000000
+last_sent: 2023-08-02T16:45:53.742417-03:00
+```
+
+The `uid` field attracts the most attention since initially an user does not know how its value is generated. The field's value is an UUID generated according to the specifications made on [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122) and [DCE 1.1](https://pubs.opengroup.org/onlinepubs/9696989899/toc.htm).
+
+## Types of telemetry data
+
+Bellow is an JSON object with an example of a full telemetry data message that is sent from the CLI to us:
+
+```json
+{
+  "uid": "00000000-0000-0000-0000-000000000000",
+  "os": {
+    "operating_system": "linux",
+    "arch": "amd64",
+    "binary_version": "3.27.0-dev",
+    "golang_version": " go1.20.7"
+  },
+  "cli": {},// An empty object instead of `null` to represent that the data is sent by a CLI.,
```

Comment:
`// An empty object for this field indicates that this data was sent by a CLI instance`.